.github/workflows/CI.yml

```
@@ -21,53 +21,54 @@ permissions:
 
 jobs:
   test-rust-build:
-    runs-on: [self-hosted, Linux, large]
+    runs-on: mco-dev-large-x64
     container: mobilecoin/builder-install:v0.0.25
     steps:
-      - name: Check out code
-        uses: actions/checkout@v3
-        with:
-          submodules: 'recursive'
-      - name: Setup Rust
-        uses: ./.github/actions/setup-rust
-      - name: Cargo build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --locked
-      - name: Check dirty git
-        uses: ./.github/actions/check-dirty-git
+    - name: Checkout
+      uses: mobilecoinofficial/gh-actions/checkout@v0
+
+    - name: Setup Rust
+      uses: ./.github/actions/setup-rust
+
+    - name: Cargo build
+      shell: bash
+      run: |
+        cargo build --release --locked
+
+    - name: Check dirty git
+      uses: ./.github/actions/check-dirty-git
+
   lint-rust:
-    runs-on: [self-hosted, Linux, large]
+    runs-on: mco-dev-large-x64
     container: mobilecoin/builder-install:v0.0.25
-
     steps:
-      - name: Check out code
-        uses: actions/checkout@v3
-        with:
-          submodules: 'recursive'
-      - name: Setup Rust
-        uses: ./.github/actions/setup-rust
-        with:
-          components: rustfmt, clippy
-      - name: Run lint script
-        uses: ./.github/actions/lint
-      - name: Check dirty git
-        uses: ./.github/actions/check-dirty-git
+    - name: Checkout
+      uses: mobilecoinofficial/gh-actions/checkout@v0
+
+    - name: Setup Rust
+      uses: ./.github/actions/setup-rust
+      with:
+        components: rustfmt, clippy
+
+    - name: Run lint script
+      uses: ./.github/actions/lint
+
+    - name: Check dirty git
+      uses: ./.github/actions/check-dirty-git
   test:
-    runs-on: [self-hosted, Linux, large]
+    runs-on: mco-dev-large-x64
     container: mobilecoin/builder-install:v0.0.25
     steps:
-      - name: Check out code
-        uses: actions/checkout@v3
-        with:
-          submodules: 'recursive'
-      - name: Setup Rust
-        uses: ./.github/actions/setup-rust
-      - name: Cargo test
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --locked
-      - name: Check dirty git
-        uses: ./.github/actions/check-dirty-git
+    - name: Checkout
+      uses: mobilecoinofficial/gh-actions/checkout@v0
+
+    - name: Setup Rust
+      uses: ./.github/actions/setup-rust
+
+    - name: Cargo test
+      shell: bash
+      run: |
+        cargo test
```

This lost the `--locked` flag. I don't think it matters as i think the build will catch it even for tests
```suggestion
        cargo test --locked